website/project/model.py

```diff
@@ -4259,15 +4261,28 @@ def _email_template_context(self, user, node, is_authorizer=False, urls=None):
 
     def _add_success_logs(self, node, user):
         src = node.registered_from
+        log_type = NodeLog.PROJECT_REGISTERED
+
+        params = {
+            'parent_node': src.parent_id,
+            'node': src._primary_key,
+            'registration': node._primary_key,
+        }
+
+        try:
+            draft = DraftRegistration.find_one(Q("registered_node", "eq", node))
+        except NoResultsFound:
+            pass
+        else:
+            if "Prereg" in draft.registration_schema.name:
```

Comment:
Please use https://github.com/CenterForOpenScience/osf.io/blob/develop/website/prereg/utils.py instead of string comparison.